scripts/docker/launch.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
should delete. this is too specific/enforces more structure. i don't see much value here yet.

Reply:
deleted